### ISSUE TITLE
Series number is wrong?

### ISSUE THREAD

**Issue by strogonoff** [relaton/relaton-data-nist#5]
Here, `NBS BH 13` is declared as belonging in series “NBS Building and Housing Reports” under number “NBS BH”:

https://github.com/relaton/relaton-data-nist/blob/7673ee91bc159b4b53d4bbb7d5ad68b85c6d87ce/data/NBS_BH_13.yaml#L218-L222

I think that’s an error, and the `number` should be “13”, while “NBS BH” should perhaps be series abbreviation?

Source: Relaton spec defines `number` as document number within the series (see [SeriesType](https://github.com/relaton/relaton-models/blob/fee9ea8205756163a1d96de47f9b105dc701d8a0/models/SeriesType.lutaml#L42-L46)).

cc @ronaldtse, @andrew2net

**Comment by ronaldtse** [comment]
@strogonoff yes you are correct here. "BH" is the series abbreviation, "13" is the number.

**Comment by ronaldtse** [comment]
@andrew2net can we integrate relaton-nist with pubid-nist, which correctly parses all entries? Thanks.

**Comment by ronaldtse** [comment]
This ticket should go to relaton-nist.
